src/C2HS/Gen/Bind.hs

```diff
@@ -3084,8 +3086,21 @@ cCompiler = unsafePerformIO $ do
       when (code /= ExitSuccess) $
         error "Failed to determine C compiler from 'ghc --info'!"
       let vals = read stdout :: [(String, String)]
-      case Prelude.lookup "C compiler command" vals of
-        Nothing -> error "Failed to determine C compiler from 'ghc --info'!"
-        Just cc -> do
-          writeIORef cCompilerRef $ Just cc
-          return cc
+      case (Prelude.lookup "C compiler command" vals, Prelude.lookup "LibDir" vals) of
+        (Just cc, Just topDir) -> do
+          -- ensure that $topdir is expanded
+          let mungedCc = mugnePath topDir cc
```

Comment:
s/`mugnePath`/`mungePath`? :)

Reply:
Yeah, the new version is up in a moment. I ran into a few issue with spaces.